MANIFEST.in

```diff
@@ -32,3 +32,5 @@ prune crt/aws-lc/tool
 prune crt/aws-lc/util
 include crt/aws-lc/util/fipstools/CMakeLists.txt
 include crt/aws-lc/util/fipstools/acvp/modulewrapper/CMakeLists.txt
+# by default only test/test*.py are included, include the entire test suite
+graft test
```

Comment:
Thanks again!
Thanks for fixing this!

But looking at the files this adds to the source dist ... it includes a [10MiB txt file](https://github.com/awslabs/aws-crt-python/blob/f6bc9fbdd55b335de0b87adae28a33864b0ce4e6/test/test_s3.py#L138) that just exists so we have a large file to upload, the contents are meaningless..

Let's not merge this until we get that file out of source control. The test can just write its own tmp file during setUp instead of keeping it in source control. Anyway, one of our people will jump on this today